cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,80 @@
+import { generateUser } from '../support/commands';
+
 /// <reference types='cypress' />
 
+const user = generateUser();
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+
+  it(' should be elements of pagination is present', () => {
```

Comment:
There is an extra space at the beginning of the test description string. It should be removed for consistency.